src/wasm/wasm-validator.cpp

```diff
@@ -185,8 +185,8 @@ struct FunctionValidator : public WalkerPass<PostWalker<FunctionValidator>> {
     };
 
     Type type;
-    Index arity;
-    BreakInfo() : arity(UnsetArity) {}
+    Index arity{UnsetArity};
```

Comment:
Is this preferred/different to `Index arity = UnsetArity;`?

Reply:
https://stackoverflow.com/a/18222927

Reply:
Hmm, good point. In that case it might be nice to change the enum to say `static const Index UnsetArity = -1;`